documentation/asciidoc/computers/configuration/pin-configuration.adoc

```diff
@@ -35,12 +33,12 @@ sudo dtc -I dts -O dtb -o /boot/firmware/dt-blob.bin dt-blob.dts
 Similarly, a `.dtb` file can be converted back to a `.dts` file, if required.
 
 ----
-dtc -I dtb -O dts -o dt-blob.dts /boot/firmware/dt-blob.bin
+dtc -I dtb -O dts -o dt-.dts /boot/firmware/dt-.bin
```

Comment:
Eeek, what happened here @nathan-contino ??

Reply:
Egad. I'm going to blame the GitHub editor for this nonsense. My bad for claiming "Indeed" to Al's question before.

Reply:
Opened https://github.com/raspberrypi/documentation/pull/3194 to fix.